### DISCUSSION TITLE
GlobalOpenTelemetry build warning

### DISCUSSION THREAD

**Discussion by Camsteack** [open-telemetry/opentelemetry-java#2916] [Q&A]
Hi, I am trying to use opentelemetry to instrument our application but when building it I am getting a warning on a missing dependency, and as we are treating warning as errors it becomes a problem for us.

```
/Users/camilleballouche/.gradle/caches/modules-2/files-2.1/io.opentelemetry/opentelemetry-api/0.17.1/f7cd56a17cb540c2d32d006865d36c873a3187e2/opentelemetry-api-0.17.1.jar(/io/opentelemetry/api/GlobalOpenTelemetry.class): warning: Cannot find annotation method 'value()' in type 'GuardedBy': class file for com.google.errorprone.annotations.concurrent.GuardedBy not found
```

If I manually add the dependency it removes the warning but I would like to avoid having to do that. Any pointer ?
Many thanks !

**Comment by anuraaga** [comment]
Hi @camsteack - you're seeing #2897 which is fixed at head so it should work for you in the next release. Sorry for the troubles.

**Reply by Camsteack** [reply]
Thanks a lot for the quick response ! Sorry I missed the existing issue, no problem I have the workaround until the next release :)

**Reply by samvaity** [reply]
Hello @anuraaga, I am still seeing the compilation error when including the error_prone compile-time dependency in my project with OTel version `1.0.0`. Could you let me know if this should be happening?

**Reply by trask** [reply]
When updating to 1.0.0, try adding `com.google.code.findbugs:jsr305` dependency (and removing `com.google.errorprone:error_prone_annotations` dependency), see https://github.com/open-telemetry/opentelemetry-java/issues/2897#issuecomment-790039252

**Reply by trask** [reply]
and thanks for letting us know about this!
